src/vue/modules/verification/general-form/countries.js

```diff
@@ -0,0 +1,9 @@
+import countries from '@/i18n/en.countries'
```

Comment:
localization files should not be the source of truth of the valid country list set, I still pref using some npm package of implementation of iso3166-2